### PR TITLE
Display links to select another CAZ only if chargeable in more than one zone

### PR DESCRIPTION
**Pull request by kingarj**
## Motivation and Context

Required by [CAZ-1666](https://eaflood.atlassian.net/browse/CAZ-1666)

## Description

Dynamically display links (specifically the 'Change' link for the LA on the check your answers page and the 'Pay for another Clean Air Zone' link on the success page) based on whether or not there are any further zones that the vehicle is chargeable in.

## How Has This Been Tested?

Two additional cucumber tests added to ensure that the links do not appear when the vehicle is only chargeable in one zone.

## Checklist:

- [x] It contains only changes required by issue (does not contain other PR)
- [x] Includes the link to an issue (if apply)
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: app/services/session_manipulation/set_chargeable_zones.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module SessionManipulation
+  ##
+  # Service used to mark vehicle as a taxi of PHV registered in Leeds.
+  #
+  # ==== Usage
+  #    SessionManipulation::SetLeedsTaxi.call(session: session)
```
Comment (miqs1992) [inline app/services/session_manipulation/set_chargeable_zones.rb]: This is not a valid example

File: app/services/session_manipulation/set_chargeable_zones.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    # ==== Attributes
+    # * +session+ - the user's session
+    # * +la_id+ - UUID, id of the Clean Air Zone
+    #
+    def initialize(session:, chargeable_zones:)
+      @session = session
+      @chargeable_zones = chargeable_zones
+    end
+
+    # Sets +leeds_taxi+ to true in the session. Used by the class level method +.call+
```
Comment (miqs1992) [inline app/services/session_manipulation/set_chargeable_zones.rb]: This is not documentation for this class

File: app/views/charges/review_payment.html.haml [hunk 3]
```diff
@@ -24,7 +24,7 @@
               %th.govuk-table__header{scope: 'row', id: 'value-la'}
                 = @la_name
               %td.govuk-table__cell
-                = link_to 'Change', local_authority_charges_path, class: 'govuk-link', id: 'change-la'
+                = link_to 'Change', local_authority_charges_path, class: 'govuk-link', id: 'change-la' if @chargeable_zones > 1
```
Comment (miqs1992) [inline app/views/charges/review_payment.html.haml]: Can you verify that this is not causing any Wave issues if the td is empty?

**Comment by kingarj** [comment]
@miqs1992 Documentation has been updated and I have checked the missing link on Wave - no problems raised.

**Pull request merged**